### DISCUSSION TITLE
CslaContrib & Csla 5.3?

### DISCUSSION THREAD

**Discussion by ajohnstone-ks** [MarimerLLC/csla#1752] [Questions] [answered]
It looks like CslaContrib hasn't been updated to work with Csla 5.3.0 and trying to use it brings in the Csla-Core package, which conflicts with Csla. 

Is this just not supported anymore?

**Comment by rockfordlhotka** [comment, marked as answer]
The project is waiting for a community leader. Tiago was leading it for years, and he completely disappeared without warning last year (I've been unable to reach him via any channel, and am quite worried).

**Reply by ajohnstone-ks** [reply]
Yikes, that is worrying.  Hope he's ok.

**Reply by BlagoCuljak** [reply]
Hi, we use CSLAContrib 5.3 in our project, the project just needs to upgrade NuGet packages, and you're good to go.

**Reply by ajohnstone-ks** [reply]
@BlagoCuljak I didn't see any updated nuget packages available.

**Reply by BlagoCuljak** [reply]
1. Download CSLAContrib project
2. Add it to your solution
3. Remove CSLAContrib NuGet references
4. Add CSLAContrib Project reference were you had NuGet packages that you have removed
5. Upgrade CSLA NuGet package in CSLAContrib project

Hope it helps.
